### PR TITLE
JSDoc Updates

### PR DESCRIPTION
**Pull request by lishaduck**
Sort, format, lint jsdocs.

Depends on #164

Here's the diff as GH doesn't play well with stacks from forks: https://github.com/lishaduck/node-elm-review/compare/turbo...jsdocs

### CODE REVIEW

File: lib/anonymize.js [hunk 1]
```diff
@@ -7,8 +7,8 @@ machines, and also the same output when only the CLI version changes.
 const path = require('path');
 
 /**
- * @typedef { import("./types/options").Options } Options
- * @typedef { import("./types/path").Path } Path
+ * @import {Options} from "./types/options"
```
Comment (jfmengels) [inline lib/anonymize.js:10]: Yay for TS 5.5 :tada:

File: vendor/node-elm-compiler.js [hunk 2]
```diff
@@ -1,9 +1,5 @@
 'use strict';
 
-/**
- * @typedef { import("../lib/types/options").Options } Options
```
Comment (jfmengels) [inline vendor/node-elm-compiler.js:4]: Is there a tool out there that you know of that can help finding these unused imports? Or do we already have one?

**Comment by jfmengels** [comment]
Thanks again. There was a TS error that was introduced here but I pushed a fix.

File: vendor/node-elm-compiler.js (hunk 2 above)
Comment (lishaduck) [reply vendor/node-elm-compiler.js:4]: TS can do it, but it's buggy, so I turned it back off after I turned it on.
microsoft/TypeScript#58969

**Comment by lishaduck** [comment]
> Thanks again ☺️ There was a TS error that was introduced here but I pushed a fix.

Thanks! I did a bunch of rebasing and must've missed one.

**Comment by jfmengels** [comment]
Yeah, I jsut noticed you fixed the issue in the typings PR. That's fine, no worries :+1:

**Pull request merged**